docs/decisions/0033-stripe-pending-updates-license-drift-prevention.rst

```diff
@@ -0,0 +1,86 @@
+0033. Stripe Pending Updates for License Count Drift Prevention
+===============================================================
+
+Status
+------
+
+Proposed - October 2025
+
+Context
+-------
+
+Enterprise customers can modify their subscription quantities (license counts) through Stripe's billing portal.
+By default, Stripe immediately applies subscription changes regardless of whether payment succeeds or fails.
+This creates a potential license count drift problem:
+
+1. Customer increases quantity from 10 to 20 licenses in billing portal
+2. Stripe immediately updates subscription to 20 licenses  
+3. Payment fails for the additional licenses
+4. Our internal systems reflect 20 licenses but payment never succeeded
+5. Customer gets access to 10 extra unpaid licenses
+
+This drift between Stripe subscription quantities and actual paid licenses poses both
+revenue risk and compliance issues.
+
+Decision
+--------
+
+We will implement Stripe's "pending updates" feature to ensure subscription
+changes only apply after successful payment:
+
+1. **Enable pending updates on subscription creation**: When a new subscription is created and we receive a
+   ``customer.subscription.created`` webhook, immediately call ``stripe.Subscription.modify()`` with
+   ``payment_behavior='pending_if_incomplete'``
+
+2. **Handle pending update lifecycle**: Add webhook handlers for:
+   - ``customer.subscription.updated`` - to track when subscriptions have pending updates
+   - ``customer.subscription.pending_update_applied`` - when payment succeeds and changes are applied
+   - ``customer.subscription.pending_update_expired`` - when payment fails and changes are discarded  
+
+The implementation ensures that:
+- Subscription quantity changes through billing portal require successful payment before being applied
+- Failed payments result in no license count changes (no drift)
+
+See: https://docs.stripe.com/billing/subscriptions/pending-updates
+
+Rationale
+---------
+
+**Why pending updates work for our use case:**
+- Stripe's billing portal is configured to allow quantity changes
+- Pending updates prevent changes from applying on payment failure
+- The ``payment_behavior='pending_if_incomplete'`` parameter persists as subscription behavior for future updates
+
+**Why we enable it post-creation:**
+- Pending updates only apply to subscription modification operations, not initial creation
+- Checkout sessions don't support ``payment_behavior`` parameter
```

Comment:
I think this is the only bullet point needed, the other two are redundant or irrelevant IMO.